normandy/selfrepair/static/js/normandy_driver.js

```diff
@@ -79,7 +129,22 @@ let Normandy = {
                 options.learnMoreMessage,
                 options.learnMoreUrl
             );
+
+            resolve(emitter);
```

Comment:
I'm concerned about the order of execution here. What happens when an even gets emitted before the `.then()` that listens to this receives this emitter? I don't think `EventEmitter`s usually queue messages until a listener is around. I'm not sure how to fix this though, except to not use the Promise, and just return an event emitter directly.